src/libraries/System.Security.Cryptography/src/System.Security.Cryptography.csproj

```diff
@@ -1292,6 +1292,7 @@
              Link="Common\System\Security\Cryptography\RSASecurityTransforms.macOS.cs" />
     <Compile Include="$(CommonPath)System\Security\Cryptography\RSAOpenSsl.cs"
              Link="Common\System\Security\Cryptography\RSAOpenSsl.cs" />
+    <Compile Include="System\Security\Cryptography\AesGcm.macOS.cs" />
```

Comment:
Is it really macOS, or would this be Apple-all?

Reply:
macOS only for now. The native Swift shim is not compiled on iOS yet since that'd need some linker parameter changes in Xamarin.

Reply:
It's really macOS only. iOS / tvOS remain unsupported due to some linker challenges with supporting old versions of iOS.